src/main/java/at/hannibal2/skyhanni/data/MiningAPI.kt

```diff
@@ -143,7 +143,7 @@ object MiningAPI {
         if (waitingForInitSound) {
             if (event.soundName in allowedSoundNames && event.pitch == 0.7936508f) {
                 val pos = event.location.roundLocationToBlock()
-                if (recentClickedBlocks.none { it.position == pos }) return
+                if (recentClickedBlocks.toList().none { it.position == pos }) return
```

Comment:
this is useless. rather change recentClickedBlocks to either be val or be immutable if you want to fix concurrent modification exceptions